### PR TITLE
Changed favicon and logo

### PR DESCRIPTION
**Pull request by brauliobolano**
# Pull Request

## Changes Made:
- Added favicon to the website
- Added new logo to the website

## Notes:
The new favicon and logo are designed to improve the branding of the website. Please let me know if any changes are required.

### CODE REVIEW

File: src/components/Navigation.jsx [hunk 1]
```diff
@@ -1,11 +1,9 @@
 export default function Navigation() {
   return (
     <header className="header-nav container">
-      <img
-        className="logo"
-        src="/omnifood-logo.png"
-        alt="Caribe Tax Solutions logo"
-      />
+      <div className="logo">
```
Comment (cabrera-carlos) [inline src/components/Navigation.jsx:4]: Why wrapping in a div?

Comment (brauliobolano) [reply src/components/Navigation.jsx:4]: Wrapping the logo image in a <div> element allows for better organization and styling of the logo component. It provides a container for the image and any additional elements or styles that may be applied to the logo.

Comment (brauliobolano) [reply src/components/Navigation.jsx]: I cleaned up the code in Navigation.jsx

**Comment by brauliobolano** [comment]
I updated the favicon and logo

**Review by cabrera-carlos — APPROVED** [review]
Amazing work. Thank you.

**Pull request merged**